docs/recipes/plugin/readme.md

```diff
@@ -0,0 +1,18 @@
+# Lighthouse plugin recipe
+
+## Contents
+- `lighthouse-plugin-preload-as` - the plugin module
+  - `package.json` - declares the plugin's entry point (`plugin.js`)
```

Comment:
should plugins include lighthouse as a peerDependency?

Reply:
> should plugins include lighthouse as a peerDependency?

yes